spyro/solvers/acoustic_wave.py

```diff
@@ -144,3 +144,10 @@ def reset_pressure(self):
             self.u_n.assign(0.0)
         except:
             warnings.warn("No pressure to reset")
+        if self.abc_status is True:
```

Comment:
When the comparison is against true, you can simply write: "if self.abc_status:". Also, the name of the variable could be something like "abc_active" to facilitate reading.

These are just suggestions for future code. It is not necessary to change anything.

Reply:
Thanks for the suggestion, abc_active does facilitate reading. I have changed it